org.gyunaev.Birdtray.json

```diff
@@ -0,0 +1,62 @@
+{
+    "app-id": "org.gyunaev.Birdtray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "birdtray",
+    "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=dri",
+      "--filesystem=~/.thunderbird:ro",
```

Comment:
I believe this just fails if the directory doesn't exist? Can you test that.

Reply:
Tested it and it works fine even when the directory doesn't exist.